pyinfra/facts/dnf.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import unicode_literals
```

Comment:
I don't think we need this file at all - the facts in `facts/yum.py` will provide these (facts are in a global namespace).

(side note for me: I think it'd make sense to rename `facts/yum.py` -> `facts/rpm.py` as it's more accurate (rpm facts vs yum/dnf operations).)

Reply:
Done.